modules/harvest/src/HarvestPlanListBuilder.php

```diff
@@ -74,9 +74,9 @@ public function buildRow(EntityInterface $entity) {
     $harvest_plan_id = $entity->get('id')->getString();
     $run_entity = NULL;
 
-    if ($run_id = $this->harvestService->getLastHarvestRunId($harvest_plan_id)) {
+    if ($run_id = $this->harvestService->runRepository->getLastHarvestRunId($harvest_plan_id)) {
       // There is a run identifier, so we should get that info.
-      $run_entity = $this->harvestRunRepository->loadEntity($harvest_plan_id, $run_id);
+      $run_entity = $this->harvestRunRepository->runStorage->load($run_id);
```

Comment:
Inject the harvestrun storage to the list builder rather than making runstorage public and accessing it here.

Reply:
Done.